web/packages/teleport/src/Integrations/IntegrationList.tsx

```diff
@@ -339,6 +339,10 @@ const IconCell = ({ item }: { item: IntegrationLike }) => {
         formattedText = 'Datadog Incident Management';
         icon = <IconContainer name="datadog" />;
         break;
+      case 'aws-identity-center':
+        formattedText = 'AWS Identity Center';
+        icon = <IconContainer name="aws" />;
+        break;
```

Comment:
The image is rendered in the integration list table:
<img width="787" alt="Screenshot 2024-10-18 at 6 21 22 PM" src="https://github.com/user-attachments/assets/c927bb5c-ab99-422a-b712-751185328497">